Examples/SnapinsSDKExample/app/build.gradle

```diff
@@ -49,17 +49,19 @@ dependencies {
   * If you only use a single Snap-In just pull in the Snap-In you need.
   * compile "com.salesforce.service:case-ui:<VersionNumber>"
   */
-  compile 'com.salesforce.service:servicesdk:210.0.0'
+  implementation "com.salesforce.service:servicesdk:$serviceSdkVersion"
+  implementation "com.salesforce.mobilesdk:SalesforceSDK:$mobilesdkVersion"
```

Comment:
Is this dependency necessary for this example?

Reply:
Removing per my comment above.